README.md

```diff
@@ -109,3 +71,52 @@ The following files will be created:
 - `index.html`: start page for the project
 - `pyscript.toml`: project metadata and config file
 - `main.py`: a "Hello world" python starter module
+
+#### Use --wrap to embed a python file OR a command string
+
+- ##### Embed a Python script into a PyScript HTML file
+
+```shell
+$ pyscript create --wrap <filename.py>
+```
+
+This will generate a project i.e. a new directory named `filename` under the current directory.
+
+Similar to the above, interactive prompts will further ask for metadata information.
+
+The following files will be created:
+
+- `index.html`: start page for the project
+- `pyscript.toml`: project metadata and config file
+- `main.py`: contains code of `filename.py`
+
+This can be overridden with the `-o` or `--output` option:
+
+```shell
+$ pyscript create --wrap <filename.py> -o <another_filename.html>
+```
+
+i.e. the HTML file created in the above directory will now be named `another_filename.html`
```

Comment:
This is not a blocker for the PR (we can fix in a follow up PR) but I'm trying to understand the behavior here. Does this mean that if the users specifices `-o`, we'll create a folder and will have everything in a new single file?

Looking at the test for this seems like, in the case above, we'd have:

* a new `another_filename` folder
* a new file `another_filename\another_filename.html`
* a new `another_filename\main.py` file
* a new `another_filename\pyscript.toml` (test is not checking that but I'm assuming)

Is that correct?

Reply:
Yeah, that's correct except that name of folder is `filename` and not `another_filename`

Reply:
ah, I see. Ok. I think it's ok for now... we can sit down and try to make a thought through decision later. Mainly I'm afraid there might be some confusing scenarios..

Reply:
Ok, merging for now...